.github/install-tools/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "install-tools"
+edition = "2021"
+version = "0.1.0"
+publish = false
+
+[dependencies]
+tar = "0.4"
+zstd = "0.13"
+
+[workspace]
```

Comment:
Minor nit: missing newline at the end (also I would put `[workspace]` at the top so it's easy to see it self-excludes from the outer workspace).

(and maybe call it `install-spirv-tools`? the new `cargo -Zscript` stuff would be great for this kind of helper, but we don't have to switch right away ofc)